data/oyepriyansh.json

```
@@ -0,0 +1,238 @@
+{
+
+	"name": "Priyansh Prajapat",
+
+	"type": "personal",
+
+	"bio": "Aspirant Full stack Web Developer",
+
+
+
+	"socials": [
+
+		{
+			"icon": "FaInstagram",
+			"url": "https://instagram.com/oyepriyansh"
+		},
+
+		{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/oyepriyansh"
+		},
+
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/oyepriyansh"
+		},
```

Please remove this trailing comma `,` here 
```suggestion
		}
```